src/libasr/codegen/asr_to_wasm.cpp

```diff
@@ -1411,12 +1422,12 @@ class ASRToWASMVisitor : public ASR::BaseVisitor<ASRToWASMVisitor> {
         for (uint32_t i = 0; i < x.n_args; i++) {
             if (x.m_args[i].m_right) {
                 this->visit_expr(*x.m_args[i].m_right);
-                wasm::emit_i32_const(m_code_section, m_al, 1);
+                this->visit_expr(*m_dims[i].m_start);
                 wasm::emit_i32_sub(m_code_section, m_al);
                 size_t jmin, jmax;
 
                 // TODO: add this flag to ASR for each array:
-                bool column_major = true;
+                bool column_major = false;
```

Comment:
This is a temporary change. We need to support the array storage ordering (row/col major) in the `ASR` soon.

Reply:
This change will break LFortran I think. That is fine, we can diverge for a bit, we have done that in the past.

@czgdp1807 when syncing with LFortran, avoid this change. We will create a general solution soon.